frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -98,14 +101,23 @@ const plugin: Plugin<ConsumedExtensions> = [
       detect: detectOCS,
     },
     flags: {
-      required: [CEPH_FLAG],
+      required: [OCS_MODEL_FLAG],
     },
   },
   {
     type: 'FeatureFlag/Custom',
     properties: {
       detect: detectRGW,
     },
+    flags: {
+      required: [MCG_FLAG],
+    },
+  },
+  {
+    type: 'FeatureFlag/Custom',
+    properties: {
+      detect: detectComponents,
```

Comment:
```suggestion
      detect: detectOCSComponents,
```

Reply:
I prefer `detectComponents`. We are detecting components from OCS Plugin.